cli/src/test/resources/ide-projects/update/project/home/.ide/ide.properties

```diff
@@ -0,0 +1,15 @@
+#********************************************************************************
+# This file contains the global configuration from the user HOME directory.
+#********************************************************************************
+
+DOCKER_EDITION=docker
+FOO=foo-${BAR}
+
+TEST_ARGS1=${TEST_ARGS1} user1
+TEST_ARGS2=${TEST_ARGS2} user2
+TEST_ARGS3=${TEST_ARGS3} user3
+TEST_ARGS7=user7
+TEST_ARGS10=user10
+TEST_ARGSb=userb
+TEST_ARGSc=${TEST_ARGS1} userc
+TEST_ARGSd=${TEST_ARGS1} userd
```

Comment:
Also Windows Line endings in mixed EOL style.
This file is IMHO pointless for your test and should be removed.